app-crypt/gpgme/gpgme-1.23.2.recipe

```diff
@@ -6,87 +6,77 @@ Currently it uses GnuPG as its backend but the API isn't restricted to \
 
-#TODO: fix gpgme-config hardcoded package paths
 PROVIDES="
-	gpgme${secondaryArchSuffix} = $portVersion compat >= 1.9
-	cmd:gpgme_json${secondaryArchSuffix} = $portVersion compat >= 1.9
-	cmd:gpgme_tool${secondaryArchSuffix} = $portVersion compat >= 1.9
-	lib:libgpgme${secondaryArchSuffix} = $libgpgmeVersion compat >= 11
-	lib:libgpgmepp${secondaryArchSuffix} = $libgpgmeppVersion compat >= 6
-	lib:libqgpgme${secondaryArchSuffix} = $libqgpgmeVersion compat >= 7
+	gpgme$secondaryArchSuffix = $portVersion compat >= 1.17
+	cmd:gpgme_json$secondaryArchSuffix = $portVersion compat >= 1.17
+	cmd:gpgme_tool$secondaryArchSuffix = $portVersion compat >= 1.17
+	lib:libgpgme$secondaryArchSuffix = $libgpgmeVersionCompat
+	lib:libgpgmepp$secondaryArchSuffix = $libgpgmeppVersionCompat
+	lib:libqgpgme$secondaryArchSuffix = $libqgpgmeVersionCompat
 	"
 REQUIRES="
-	haiku${secondaryArchSuffix}
-	lib:libassuan${secondaryArchSuffix}
+	haiku$secondaryArchSuffix
+	lib:libassuan$secondaryArchSuffix
 	lib:libglib_2.0$secondaryArchSuffix
-	lib:libgpg_error${secondaryArchSuffix}
-	lib:libQt5Core${secondaryArchSuffix}
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
-	gpgme${secondaryArchSuffix}_devel = $portVersion compat >= 1.9
-	cmd:gpgme_config${secondaryArchSuffix} = $portVersion compat >= 1.9
-	devel:libgpgme${secondaryArchSuffix} = $libgpgmeVersion compat >= 11
-	devel:libgpgmepp${secondaryArchSuffix} = $libgpgmeppVersion compat >= 6
-	devel:libqgpgme${secondaryArchSuffix} = $libqgpgmeVersion compat >= 7
+	gpgme${secondaryArchSuffix}_devel = $portVersion compat >= 1.17
+	devel:libgpgme$secondaryArchSuffix = $libgpgmeVersionCompat
+	devel:libgpgmepp$secondaryArchSuffix = $libgpgmeppVersionCompat
+	devel:libqgpgme$secondaryArchSuffix = $libqgpgmeVersionCompat
 	"
 REQUIRES_devel="
-	gpgme${secondaryArchSuffix} == $portVersion base
-	devel:libassuan${secondaryArchSuffix}
+	gpgme$secondaryArchSuffix == $portVersion base
+	devel:libassuan$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libassuan${secondaryArchSuffix}
+	devel:libassuan$secondaryArchSuffix
 	devel:libglib_2.0$secondaryArchSuffix
-	devel:libgpg_error${secondaryArchSuffix}
-	devel:libQt5Core${secondaryArchSuffix}
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:autoconf
-	cmd:automake
-	cmd:awk
-	cmd:bison
-	cmd:gcc${secondaryArchSuffix}
-	cmd:git
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
 	cmd:gpg$secondaryArchSuffix
 	cmd:gpgconf$secondaryArchSuffix
 	cmd:gpg_error$secondaryArchSuffix
-	cmd:grep
-	cmd:kill
-	cmd:ld${secondaryArchSuffix}
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:makeinfo
 	cmd:msgfmt
-	cmd:perl
-	cmd:ps
 	cmd:pkg_config$secondaryArchSuffix
-	cmd:qmake$secondaryArchSuffix >= 5.12
-	cmd:tar
-	cmd:xargs
 	"
 
 BUILD()
 {
 	autoreconf -fi
 	runConfigure ./configure
 	make $jobArgs
+
```

Comment:
Not sure about the next phrase, it runs fine the first build but fails on a second build (figures as gpg-agent is already nuked) ...